autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/SecureStoreBackedAutofillStore.kt

```diff
@@ -270,7 +270,7 @@ class SecureStoreBackedAutofillStore @Inject constructor(
         return matchType
     }
 
-    private fun LoginCredentials.prepareForReinsertion(): WebsiteLoginDetailsWithCredentials {
+    private fun LoginCredentials.prepareForBulkInsertion(): WebsiteLoginDetailsWithCredentials {
```

Comment:
reusing this for bulk inserting, so renaming so it's no longer specific to re-insertion